aws/src/main/java/org/apache/iceberg/aws/s3/S3URI.java

```diff
@@ -31,7 +31,7 @@
  *
  * <p>Note: Path-style access is deprecated and not supported by this implementation.
  */
-class S3URI {
+public class S3URI {
```

Comment:
do we need to make this public?

Reply:
For using in `analayticsaccelerator` package I made it public earlier. Now I've moved the relevant code to s3 package and avoided making it public.